### PR TITLE
Disable use of structs in shaders by default

### PR DESCRIPTION
**Pull request by levinli303**
I think it is better to disable it by default for compatibility

### CODE REVIEW

**Comment by 375gnu** [comment]
@levinli303 Could you add another commit which adds this definition via cmake?

**Review by 375gnu — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -64,14 +64,20 @@ option(ENABLE_GLES          "Build for OpenGL ES 2.0 instead of OpenGL 2.1 (Defa
 option(USE_GTKGLEXT         "Use libgtkglext1 for GTK2 frontend (Default: on)" ON)
 option(USE_QT6              "Use Qt6 in Qt frontend (Default: off)" OFF)
 option(USE_GTK3             "Use Gtk3 in GTK2 frontend (Default: off)" OFF)
-option(USE_WAYLAND          "Use Wayland in Qt frontend (Default: off)" OFF)
```
Comment (munix9) [inline CMakeLists.txt:67]: USE_WAYLAND option is gone again?

Comment (levinli303) [reply CMakeLists.txt:67]: oops, I think I’ve deleted this line by accident.

Comment (levinli303) [reply CMakeLists.txt:67]: I don’t have access to a PC now, so if anyone can help add it back, feel free to do so.

Comment (375gnu) [reply CMakeLists.txt:67]: oops! :) i returned it back

**Pull request merged**